src/solver.jl

```diff
@@ -63,8 +85,10 @@ Both summation-by-parts operators should be associated with the same grid.
 """
 function Solver(D1::AbstractDerivativeOperator{RealT},
                 D2::Union{AbstractDerivativeOperator{RealT}, AbstractMatrix{RealT},
-                          Nothing}) where {RealT}
-    Solver{RealT, typeof(D1), typeof(D2)}(D1, D2)
+                          Nothing} = nothing,
```

Comment:
With the new default of `nothing` for `D2` could you please replace [all occurrences of `solver(D1, nothing)`](https://github.com/search?q=repo%3ANumericalMathematics%2FDispersiveShallowWater.jl%20Solver(D1%2C%20nothing)&type=code) by `solver(D1)`?